src/app/App.tsx

```diff
@@ -34,11 +34,38 @@ const Main = () => {
 
 	const isOnline = useNetworkStatus();
 
+	const syncDelegates = (env: Environment) => {
+		console.log("Running delegates sync...");
+		const coinsData = env.usedCoinsWithNetworks();
+		const coinsInUse = Object.keys(coinsData);
+		const delegatesPromises: any = [];
+
+		coinsInUse.forEach((coin) => {
+			const coinNetworks = coinsData[coin];
+			coinNetworks.forEach((network) => {
+				delegatesPromises.push(Promise.resolve(env.coins().syncDelegates(coin, network)));
+			});
+		});
+
+		Promise.allSettled(delegatesPromises).then((results: any) => {
+			results.forEach(({ status }: { status: string; value: any }) => {
```

Comment:
Always use `for (const of )` instead `forEach` for consistency with all other code.